src/test/java/PageFactory/MainPage.java

```diff
@@ -0,0 +1,64 @@
+package PageFactory;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class MainPage extends BaseStepPage {
```

Comment:
You have one page, but this is not truth. Have at least 2: login page and home page